protocol-designer/src/load-file/migration/8_2_0.ts

```diff
@@ -0,0 +1,157 @@
+import { OT2_ROBOT_TYPE } from '@opentrons/shared-data'
+import { PAUSE_UNTIL_TIME } from '../../constants'
+
+import type { ProtocolFile } from '@opentrons/shared-data'
+import type { DesignerApplicationData } from './utils/getLoadLiquidCommands'
+
+const getTimeFromIndividualUnits = (
+  seconds: any,
+  minutes: any,
+  hours?: any
+): string => {
+  const hoursString = hours !== undefined ? `${hours ?? 0}:` : ''
+  return `${hoursString}${minutes ?? 0}:${seconds ?? 0}`
+}
+
+export const migrateFile = (
+  appData: ProtocolFile<DesignerApplicationData>
+): ProtocolFile<DesignerApplicationData> => {
+  const { designerApplication } = appData
+
+  if (designerApplication == null || designerApplication?.data == null) {
+    throw Error('The designerApplication key in your file is corrupt.')
+  }
+
+  const savedStepForms = designerApplication.data
+    ?.savedStepForms as DesignerApplicationData['savedStepForms']
+
+  const savedStepsWithConsolidatedTimeField = Object.values(
+    savedStepForms
+  ).reduce((acc, form) => {
+    if (form.stepType === 'pause') {
+      const {
+        id,
+        pauseHour,
+        pauseMinute,
+        pauseSecond,
+        pauseTime,
+        pauseAction,
+      } = form
+      const pauseFormIndividualTimeUnitsRemoved = Object.keys(
+        form as Object
```

Comment:
can this be typed? I think PauseForm works right?

```suggestion
        form as PauseForm
```

Reply:
I guess PauseForm is the new type though so it doesn't include the old fields that we are deprecating. i guess it is ok to just use `Object` here